packages/ui/components/ui/tx/actions-views/swap/one-way-swap.tsx

```diff
@@ -10,15 +11,35 @@ import { getAmount } from '@penumbra-zone/getters/value-view';
  * 1.23INPUT -> 4.56OUTPUT
  */
 export const OneWaySwap = ({ input, output }: { input: ValueView; output: ValueView }) => {
-  const outputAmount = getAmount.optional(output);
+  // const outputAmount = getAmount.optional(output);
 
   return (
     <div className='flex items-center gap-2'>
-      <ValueViewComponent view={input} />
+      <Density compact>
+        <div className='ml-4'>
+          <ValueViewComponent
+            valueView={input}
+            context='default'
+            priority='primary'
+            hideSymbol={true}
+            abbreviate={false}
+          />
+        </div>
+      </Density>
 
       <ArrowRight />
 
-      <ValueViewComponent view={output} showValue={!!outputAmount} />
+      <Density compact>
+        <div className='ml-4'>
+          <ValueViewComponent
```

Comment:
set `showValue` prop to `false` because the [prepaid claim fee](https://github.com/penumbra-zone/web/issues/889) can't be shown in the public view